include/featured.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+function getFeaturedPosts(){
+    $result = dbQuery('
+        SELECT *
+        FROM blogPosts
+        WHERE isFeatured = 1
+        AND delete_at is NULL
+        ORDER BY datePublished DESC
+        
+    ');
+    return $result;
+}
+
+function updateFeaturedPosts($blogPostId, $categoryId){
+    $result = dbQuery('
+        UPDATE blogPosts
+        SET isFeatured = 1 
+        WHERE blogPostId = :blogPostId
+        AND categoryId = :categoryId
+    ',array(
+        'blogPostId' => $blogPostId
+    ))->fetch();
+    return $result;
+
+}
+
+function deleteFeaturedPost($blogPostId){
+    $result = dbQuery('
+        UPDATE blogPosts
+        SET delete_at = CURRENT_TIMESTAMP
```

Comment:
I have it so that if a post on the featured page has a delete_at timestamp then it doesn't show, but it doesn't actually delete the post from the database. That way the post still shows up elsewhere on the website.

Reply:
It seems to me like you'd just want to unset that the post was featured. Would be cleaner in the db, and I could see someone (including you in the future) misinterpreting what `delete_at` means. The only reason to have this field in the db is if it's important to you to know exactly when a featured post was taken down. If you need that for some reason, I'd rename this database field to say `timeRemovedAsFeaturesPost` or something more descriptive.